federation-js/src/composition/types.ts

```diff
@@ -39,7 +39,7 @@ export interface FederationField {
   requires?: ReadonlyArray<SelectionNode>;
   provides?: ReadonlyArray<SelectionNode>;
   belongsToValueType?: boolean;
-  appliedDirectives?: DirectiveNode[]
+  otherKnownDirectives?: DirectiveNode[]
```

Comment:
Note that these are directive usages / applications, not directive definitions. (The naming in `graphql-js` is confusing, but `GraphQLDirective` is a directive definition, while `DirectiveNode` is a usage.)

Reply:
Got it! Went through and clarified the naming a bit further - everything is either named usage or definition accordingly. A bit verbose but helpful I think.